Web/ForgeTrust.AppSurface.Docs/wwwroot/css/app.css

```
@@ -602,6 +613,150 @@ body {
+    cursor: pointer;
+    font: inherit;
+    line-height: 1;
+    opacity: 0.78;
+    transition: color 140ms ease, border-color 140ms ease, background-color 140ms ease, box-shadow 140ms ease, opacity 140ms ease;
+}
+
+.docs-section-copy-icon {
+    width: 1rem;
+    height: 1rem;
+    fill: none;
+    stroke: currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)